ms/applink.c

```diff
@@ -34,9 +34,18 @@
 #define APPLINK_MAX     22      /* always same as last macro */
 
 #ifndef APPMACROS_ONLY
+
+/*
+ * Normally, do not define APPLINK_NO_INCLUDES.  Define it if you are using
+ * symbol preprocessing and do not want the preprocessing to affect the
+ * following included header files.  You will need to put these
+ * include lines somewhere in the file that is including applink.c.
+ */
+#ifndef APPLINK_NO_INCLUDES
 # include <stdio.h>
 # include <io.h>
 # include <fcntl.h>
+#endif
```

Comment:
We have bad preprocessor macro indentation here. Everything should be shifted by one space to right.